src/bidiMapper/domains/network/NetworkUtils.ts

```diff
@@ -21,7 +21,11 @@
  */
 import type {Protocol} from 'devtools-protocol';
 
-import type {Network} from '../../../protocol/protocol.js';
+import {
+  InvalidArgumentException,
+  UnsupportedOperationException,
+} from '../../../protocol/ErrorResponse';
```

Comment:
We need the rule that forces the use of extensions as this may brake in Puppeteer under some cases.

Reply:
Good point, we should always add an extension for running in the browser

Reply:
missed it. Automatic check would be awesome